localstorage.js

```diff
@@ -0,0 +1,23 @@
+const userName = document.getElementById('name');
+const userEmail = document.getElementById('mail');
+const userMsg = document.getElementById('message');
+// storing users'Data in the local storage
+const populateLocalStorage = () => {
+  const formData = {
+    userName: userName.value,
+    userEmail: userEmail.value,
+    userMsg: userMsg.value,
+  };
+  window.localStorage.setItem('userData', JSON.stringify(formData));
+};
+// Retrieving the stored data in local storage
+const fetchData = () => {
+  const storageData = JSON.parse(window.localStorage.getItem('userData'));
+  userName.value = storageData.userName;
+  userEmail.value = storageData.userEmail;
+  userMsg.value = storageData.userMsg;
+};
+fetchData();
+userName.addEventListener('change', populateLocalStorage);
+userEmail.addEventListener('change', populateLocalStorage);
+userMsg.addEventListener('change', populateLocalStorage);
```

Comment:
- [ ] The form data doesn't get saved to `localStorage`. Please ensure it does as it is an important requirement 🙏 
- [ ] When the page loads, an error is logged to the console because you're looking for data that doesn't exist in the `localStorage`. Please fix this as well 👇 
![image](https://user-images.githubusercontent.com/46839548/189422724-bd50bc8a-57e4-4353-8a3e-947565184671.png)